cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/AbstractCruiseControlResponse.java

```diff
@@ -46,4 +46,9 @@ public void discardIrrelevantResponse(CruiseControlParameters parameters) {
       }
     }
   }
+
+  public String cachedResponse(CruiseControlParameters parameters) {
```

Comment:
Does this imply that we can only support responses that extend `AbstractCruiseControlResponse`?

Reply:
so far we don't have responses directly implementing `CruiseControlResponse` interface (except one used in test case), will this change?